bin/xdmod-fed-blade-etl

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Figure out how to use the xdmod paths using the php linker
+# turn these into sub commands maybe with something like:
+# https://github.com/basecamp/sub
```

Comment:
"a delicious way to organize programs" Delicious? really?

Reply:
yeah the one I worked with was [tj/commander.js](https://github.com/tj/commander.js)